app/controllers/api/base_controller/renderer.rb

```diff
@@ -540,6 +546,47 @@ def render_options(resource, data = {})
         klass = collection_class(resource)
         render :json => OptionsSerializer.new(klass, data).serialize
       end
+
+      def virtual_attributes_for(klass)
+        validate_attr_selection(klass).last.reject { |vattr| vattr == "href_slug" }
+      end
+
+      def determine_include_for_find(klass)
+        return nil unless klass.respond_to?(:reflect_on_association)
+
+        type             = @req.subject
+        include_for_find = []
+
+        virtual_attributes_for(klass).each do |vattr|
+          attr_name, attr_base = split_virtual_attribute(vattr)
+          next if attr_base.blank?
+          next if attr_base.include?(".") # FIXME: Allow nested relations
+          next if virtual_attribute_accessor(type, attr_name)
+          next if Rbac::Filterer::CLASSES_THAT_PARTICIPATE_IN_RBAC.include?(attr_base)
```

Comment:
This is just ensuring we aren't circumventing any Rbac checks.  If the `attr_base` participates in `Rbac`, then skip it and let it be fetched normally.